infra/kdp-core/template.cue

```diff
@@ -127,6 +131,13 @@ output: {
 					}
 				]
 			},
+			if parameter.ingress.class == "traefik" {
```

Comment:
Same as above comment, if we move the judgement into middleware.cue, we don't need a judgement here.
Also, this component should be moved into the next 'step-group', next to 'apply-cloudtty-ingress'.